cfme/cloud/tenant.py

```diff
@@ -316,6 +317,23 @@ def delete(self, *tenants):
         # TODO: Assert deletion flash message for selected tenants
         # it is not shown in current UI, so not asserting
 
+    def all(self):
+        provider = self.filters.get('provider')  # None if no filter, need for entity instantiation
+        view = navigate_to(self, 'All')
+        result = []
+        for _ in view.entities.paginator.pages():
+            tenants = view.entities.get_all()
+            for tenant in tenants:
+                if provider is not None:
+                    if tenant.data['cloud_provider'] == provider.name:
+                        entity = self.instantiate(tenant.data['name'],
+                                                  get_crud_by_name(tenant.data['cloud_provider']))
```

Comment:
Required:
@lina-nikiforova you have a `provider object` in your hand, so no need to create it with a name.